tools/spelling/spelling_dictionary.txt

```diff
@@ -353,6 +353,7 @@ WRR
 WS
 WSA
 WSABUF
+WSAEINVAL
```

Comment:
Please put this in backticks in the comment; spell checker will ignore and you won't have to add it to the dictionary

Reply:
@ggreenway - author is on holiday, I'd rather not idle this 2 weeks if it is otherwise. Could we merge and in spite of this new prereq?

I suspect a wholesale refactoring of all one-and-two-off spelling terms in a new PR is a good path forward here. Since Windows porters are responsible for many, we are certainly happy to take that on as a new item. See https://github.com/envoyproxy/envoy/issues/12807

Reply:
Yes, that seems fine. As soon as we can get CI to pass.